applications/MeshingApplication/custom_utilities/mmg_utility.h

```diff
@@ -1050,6 +1058,11 @@ class MmgUtility
             }
             else
             {
+                if (mFramework == Lagrangian)
```

Comment:
really not clear to me.

kratos nodes have X() and X0() as coordinates, related as
X() = X0() + node.GetSolutionStepValue(DISPLACEMENT_X)
what is being done here? you need to interpolate BOTH, X, X0 (and the DISPLACEMENT)